pkg/apis/authenticator.aws/v1alpha1/types.go

```diff
@@ -0,0 +1,30 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:noStatus
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// IAMIdentity defines the base parsing for Identities
+type IAMIdentity struct {
```

Comment:
Should this be IAMIdentityMapping?  I think it is a little more clear but adds more typing too.  Anyone have thoughts on this?

Reply:
Hmm, would mapping be very k8s like? Thinking about other resources they're more geared toward the resource itself. Would be my 2cents but it would be good to get other opinions as well.

Reply:
Is the resource an IAMIdentity?  Or is it a IdentityMapping?  It maps one Identity IAM to a K8s Identity (username/groups).  I am open to ideas @nckturner ?  Thoughts?